### PR TITLE
Use Apache Commons in GeoJsonParser; Fix GeoJsonExtent

### PR DESCRIPTION
**Pull request by scolsen**
**N.B.**: Includes changes from pr #390 (wait until that's submitted)

- Fix an error in handling of coordinate values in GeoJsonExtent (coordinates were nested more deeply in an array in the source than I anticipated)
- Use Apache Commons to read in Json data.﻿

### CODE REVIEW

File: gnd/build.gradle [hunk 1]
```diff
@@ -207,6 +208,10 @@ dependencies {
 
     implementation "com.uber.autodispose:autodispose-android:$project.autoDisposeVersion"
     implementation "com.uber.autodispose:autodispose-android-archcomponents:$project.autoDisposeVersion"
+
+    // Apache commons IO
```
Comment (gino-m) [inline gnd/build.gradle]: Nit: Commons -> proper noun

File: gnd/src/main/java/com/google/android/gnd/persistence/geojson/GeoJsonParser.java [hunk 2]
```diff
@@ -42,34 +40,24 @@
...
-  }
-
   /**
    * Returns the immutable list of tiles specified in {@param geojson} that intersect {@param
    * bounds}.
    */
   public ImmutableList<Tile> intersectingTiles(LatLngBounds bounds, File file) {
     try {
-      String fileContents = readJsonFile(file);
+      String fileContents = FileUtils.readFileToString(file, Charset.forName("UTF-8"));
```
Comment (gino-m) [inline gnd/src/main/java/com/google/android/gnd/persistence/geojson/GeoJsonParser.java]: Nit: Please extract Charset to constant.

File: gnd/build.gradle (hunk 1 above)
Comment (scolsen) [reply gnd/build.gradle]: done!

File: gnd/src/main/java/com/google/android/gnd/persistence/geojson/GeoJsonParser.java (hunk 2 above)
Comment (scolsen) [reply gnd/src/main/java/com/google/android/gnd/persistence/geojson/GeoJsonParser.java]: Done!

**Pull request merged**